zjit/src/hir.rs

```diff
@@ -4304,6 +4378,8 @@ mod validation_tests {
     fn instruction_appears_twice_with_different_ids() {
         let mut function = Function::new(std::ptr::null());
         let entry = function.entry_block;
+        let jit_entry = function.jit_entry_block;
+        function.push_insn(jit_entry, Insn::Jump(BranchEdge { target: entry, args: vec![] }));
```

Comment:
entry -> jit entry

Reply:
ditto https://github.com/ruby/ruby/pull/14643#discussion_r2377068160

Reply:
addressed this in https://github.com/ruby/ruby/pull/14643/commits/99655554130a6fd95d9aa220fae8977e73d67e7d